IDE/HEXIWEAR/wolfSSL_HW/.gitignore

```diff
@@ -0,0 +1 @@
+Debug/
```

Comment:
Recommend adding IDE/HEXIWEAR/wolfSSL_HW/Debug to the root .gitignore and removing this one.